interface/modules/custom_modules/oe-module-weno/sql/table.sql

```diff
@@ -0,0 +1,89 @@
+#IfTable weno_pharmacy
```

Comment:
i wouldn't remove tables like this or else risk losing all data if somebody runs this script after install/use
Is there a reason this table needs to be dropped? (ie. does it already exist in OpenEMR or some other reason?)

Reply:
No, it doesnt exist in the OpenEMR codebase. i'll remove the drop statement